rocksdb_admin/admin_handler.h

```diff
@@ -130,6 +128,12 @@ class AdminHandler : virtual public AdminSvIf {
   std::unique_ptr<rocksdb::DB> removeDB(const std::string& db_name,
                                         AdminException* ex);
 
+  // Kafka consumer related helper functions
+  int getPartitionIDFromDBName(const std::string& db_name);
+  std::string getHostName();
+  std::string getConsumerGroupId(const std::string& topic_name,
+      const std::string& partition_id);
```

Comment:
make these 3 functions static? or move them into the anonymous namespace in admin_handler.cpp.

I slightly perfer the second approach, which will make the admin handler class interface less complicated.

Reply:
Used the second approach to keep it consistent with existing function.